server/citygml/attributes.go

```
@@ -0,0 +1,654 @@
+}
+
+func (ae *attributeExtractor) gmlID(attr []xml.Attr) string {
+	for _, a := range attr {
+		if ae.ns[a.Name.Space] == "http://www.opengis.net/gml" && a.Name.Local == "id" {
+			return a.Value
+		}
+	}
+	return ""
+}
+
+func (ae *attributeExtractor) getLOD(e xml.StartElement) (int, error) {
```

golangci-lint (1.62.2)

func `(*attributeExtractor).getLOD` is unused

(unused)